packages/governance/test/swingsetTests/contractGovernor/governedContract.js

```diff
@@ -38,17 +44,20 @@ const start = async (zcf, privateArgs) => {
   const { initialPoserInvitation } = privateArgs;
 
   const paramManager = await makeParamManager(
+    {
+      [MALLEABLE_NUMBER]: ['nat', numberParam.value],
```

Comment:
I still don't like using strings here.  And WebStorm doesn't provide the assistance that VSCode does.  It has no understanding of the constraints. Is it possible to use type names instead of strings here?

![NoHelp](https://user-images.githubusercontent.com/13372636/158265502-ca2fcd57-45f8-45e1-8e6b-f93f4102cf0e.png)
 
Do you know what info VSCode is keying off of here? WebStorm has access to the same language server that VSCode uses.

Reply:
I figured that VSCode is using the tsc's [LSP](https://microsoft.github.io/language-server-protocol/), though I don't know why WebStorm wouldn't also be. 

I'll make string exports from typedParamManager so these can be manifest

Reply:
Oh, I wonder if I just have to kick the language server. I'll take another look at that...

That didn't work, but telling WebStorm that `.js` means `typescript` worked. I'll expect more type checking help from WebStorm from here out.

Reply:
Glad we got that sorted out. I'll leave the strings since the type checker will verify they're legit and both our IDEs will autocomplete them.

Reply:
Dean added another reason to prefer non-strings. All the refactoring support can work with names, and distinguish the same string as being from separate namespaces.  So if 'nat' here is `GOVERNED_PARAM.NAT`, then refactoring can distinguish it from `NAT` in some other context.